tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -99,9 +168,16 @@ def test_delete_discovered_host_1(self):
 
         @Assert: Selected host should be removed successfully
 
-        @Status: Manual
-
         """
+        mac = gen_mac(multicast=True, locally=True)
+        hostname = 'mac{0}'.format(mac.replace(':', ""))
+        self._pxe_boot_host(mac)
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            self.navigator.go_to_discovered_hosts()
+            self.assertIsNotNone(self.discoveredhosts.search(hostname))
+            self.discoveredhosts.delete(hostname)
+            self.assertIsNone(self.discoveredhosts.search(hostname))
```

Comment:
Please, take a look on https://github.com/SatelliteQE/robottelo/pull/2809. I think it will be submitted really soon. In that case, you will need to remove that statement and re-test delete operation